pkg/storage/db.go

```diff
@@ -9,8 +9,8 @@ import (
 	"sync"
 	"time"
 
-	"github.com/asecurityteam/asset-inventory-api/pkg/domain"
-	_ "github.com/lib/pq" // must remain here for sql lib to find the postgres driver
+	"github.com/asecurityteam/asset-inventory-api/pkg/domain" // must remain here for sql lib to find the postgres driver
```

Comment:
Nit: comment appears to be duplicated from the pq package below

Reply:
Thank you for telling